documentation/app/templates/index.hbs

```diff
@@ -0,0 +1,96 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+}}
+
+<PageHeader @title="Overview" />
+
+<Hds::Tabs as |T|>
+  <T.Tab>About</T.Tab>
+  <T.Tab>Support</T.Tab>
+  <T.Tab>Contributing</T.Tab>
+  <T.Panel>
+    <p>Consul UI Toolkit is a collection of presentational components and
+      utilities specifically meant to be shared across the different Consul
+      applications. The toolkit contains generic components that you may have
+      come to expect from a component library, as well as very feature specific
+      components meant for the Consul apps</p>
+
+    <p>While the toolkit is an open-source project, it is intentionally built
+      for Consul products. However, it may be used outside of them. Please refer
+      to the
+      <Hds::Link::Inline
+        @href="https://github.com/hashicorp/consul-ui-toolkit/blob/main/LICENSE"
+      >License</Hds::Link::Inline>
+      before using the toolkit in your own projects.</p>
+
+    <h2>Compatibility</h2>
+    <ul>
+      <li>Ember.js v3.28 or above</li>
+      <li>Embroider or ember-auto-import v2</li>
+    </ul>
+
+    <h2>Installation</h2>
+    <CodeBlock @code="ember install @hashicorp/consul-ui-toolkit" />
```

Comment:
could we use the copy code block like they use on the helios and developer sites? If not / would be too much LOE right now we could use the list item copy component and apply style overrides OR we could just add border radius to this and add a button to copy.

<img width="1111" alt="image" src="https://user-images.githubusercontent.com/91899352/233191359-b39004b5-b15f-49f9-b957-9f0410673287.png">